bundle/compliance/cis_gcp/rules/cis_4_5/rule.rego

```diff
@@ -0,0 +1,18 @@
+package compliance.cis_gcp.rules.cis_4_5
+
+import data.compliance.lib.assert
+import data.compliance.lib.common
+import data.compliance.policy.gcp.compute.assess_instance_metadata as audit
+import data.compliance.policy.gcp.data_adapter
+
+# Ensure ‘Enable Connecting to Serial Ports’ Is Not Enabled for VM Instance
+finding = result {
+	# filter
+	data_adapter.is_compute_instance
+
+	# set result
+	result := common.generate_result_without_expected(
+		common.calculate_result(assert.is_false(audit.is_instance_metadata_valid("serial-port-enable", "true"))),
```

Comment:
nit: just for readability, can we set the audit result separately and send it to `common.calculate_result`

Reply:
same for the other rules